### ISSUE TITLE
Add additional links to upload page

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#5728]
On the profile page, there should be a message stating that the user has no packages, and if the current user is that user, a link to the upload page

On the manage packages page, there should be an "upload" button similar to the "add new organization" button

**Comment by anangaur** [comment]
![image](https://user-images.githubusercontent.com/14800916/38215706-86b19198-367d-11e8-93e9-2cdd88c50661.png)


![image](https://user-images.githubusercontent.com/14800916/38215709-895dcccc-367d-11e8-9160-c7025c0ef253.png)

**Comment by agr** [comment]
There is already an upload link on the page (And it's in direct vicinity of the proposed one). Why do we need another one?

**Comment by loic-sharma** [comment]
@anangaur The pages' content looks empty. I think it'd look better if the page's content was some image followed by a "call-to-action" link (like "Upload a new package", or "Create a new organization")

**Comment by anangaur** [comment]
@agr Many users do not expect "Upload" as a menu item and did not discover that its a good idea to show all the CRUD operations at one place on the list objects page :)

**Comment by anangaur** [comment]
@loic-sharma I agree we do not have a blank page filler for the packages page. Created an issue: https://github.com/NuGet/NuGetGallery/issues/6031
